src/utils/mockedData.ts

```diff
@@ -39,48 +124,55 @@ const regular: PaymentRequest = {
       email: 'contact@nofrixion.com',
     },
   ],
-  pispRecipientReference: 'pispOJ7SRwlX2wg75',
-  status: PaymentResult.None,
-  hostedPayCheckoutUrl: 'https://api-dev.nofrixion.com/pay/fa14171f-5fe6-4326-8c09-a9b59bbf6e7b',
-  partialPaymentMethod: PartialPaymentMethods.None,
-  inserted: new Date('2023-05-17T19:02:37.8484876+00:00'),
-  insertedSortable: '2023-05-17 19:02:37Z',
-  lastUpdated: new Date('2023-05-17T19:02:37.8475289+00:00'),
-  useHostedPaymentPage: false,
+  status: 'paid',
+  createdAt: new Date('2023-05-17T19:02:37.8484876+00:00'),
   tags: [],
+  contact: {
+    name: 'John Doe',
+    email: 'johndoe@email.com',
+  },
+  hostedPayCheckoutUrl: 'https://api-dev.nofrixion.com/pay/fa14171f-5fe6-4326-8c09-a9b59bbf6e7b',
+  description:
+    'Curabitur ultricies ligula vitae tellus fringilla consequat. Pellentesque in tortor eu nibh lobortis ultrices vel in quam. Nunc tristique egestas purus et hendrerit.',
+  productOrService: 'Flight lessons',
+  paymentAttempts: mockPaymentAttempts,
 };
 
-const noShippingAddress: PaymentRequest = {
+const partiallyPaidPaymentRequest = (): LocalPaymentRequest => {
+  const partiallyPaid: LocalPaymentRequest = { ...regular };
+  partiallyPaid.paymentAttempts = partiallyPaidMockPaymentAttempts;
+  partiallyPaid.status = 'partial';
+  return partiallyPaid;
+};
```

Comment:
You can get the same output without the need for a function.

Consider this plain old object as an example. You can easily adapt it to your situation:

```jsx
const paymentRequest = {
  id: 1,
  status: "paid",
  name: "🤘🏻",
}
```

If you want to change the get the same object but with a different status in a different variable, you'd do the following:
```jsx
const unpaidPaymentRequest = {
  ...paymentRequest, // This is your base object
  status: "unpaid", // This is the property you want to switch up
  // Fancy making more changes? Just add them here
}
```

Reply:
Wow! This is so much easier. Thanks Pablo!

Reply:
Done.